tests/conftest.py

```diff
@@ -1172,7 +1234,8 @@ def _order(
             else fulfillment_parameters
         )
 
-        lines = lines_factory(deployment_id=deployment_id) if lines is None else lines
+        asset = assets[0] if assets is not None else None
```

Comment:
Why do we pass a sequence of assets and then gets the first one instead of passing single asset?